test/cases/api.sh

```diff
@@ -1205,21 +1203,19 @@ TOKEN="$(curl localhost:8081/token | jq -r .access_token)"
 
 [ "$(curl \
         --silent \
-        --cacert /etc/osbuild-composer/ca-crt.pem \
         --output /dev/null \
         --write-out '%{http_code}' \
         --header "Authorization: Bearer $TOKEN" \
-        https://localhost/api/composer/v1/version)" = "200" ]
+        http://localhost:443/api/composer/v1/version)" = "200" ]
```

Comment:
Not port 80?

Reply:
We'd need a 2nd socket file for this. Happy to do that, we'd remove the tls config option.

Reply:
Right, and we want to drop these systemd sockets and just move to always use containers in our deployments/tests anyway, so I don't think it is worth adding a second socket which we will then drop again soon anyway... We should create a ticket to change the tests to actually test the containers we build and push to quay, but for now this is good enough I think.